data/sites.json

```diff
@@ -67,5 +67,56 @@
         "zip": "01550",
         "website": "https://signup.com/api/events?accesskey=6775657374&activity_id=3500438&enddate=2021%2F12%2F31&include_comments=false&include_jobassignments=true&include_jobs=true&my_jobs=false&selected_activity=3500438&startdate=2020%2F12%2F16",
         "signUpLink": "https://forms.gle/tMDnRLcXzZnPUxqP9"
+    },
+    "PriceChopper": {
+        "name": "Price Chopper",
+        "street": "153 Chestnut Street",
+        "city": "Southbridge",
+        "zip": "01440",
+        "websiteRoot": "https://scrcxp.pdhi.com/ScreeningEvent/e047c75c-a431-41a8-8383-81613f39dd55/GetLocations/",
+        "signUpLink": "https://www.pricechopper.com/covidvaccine/massachusetts/",
+        "locations": [{
+            "id": "1",
```

Comment:
For Hannaford you're using the zip code for the results obj `results[loc.zip]` but there could be more than one store in a zip. I suppose we could use the array key?

Reply:
Yeah I don't like what I'm doing with Hannaford. The reason it works like that is because I wasn't able to parse out if there's availability, only if there's NOT availability. So I was able to map back via the zip. If I were able to get the actual availability then I would have to change how that works.

Reply:
Using the array key seems to work.